end_to_end/test_determinism_v4_16_two_slice.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+RUN_NAME=${1}_$(date +%Y-%m-%d-%H)
+OUTPUT_PATH=${2}
+DATASET_PATH=${3}
+
+
+#Setup and Train
+#bash setup.sh
```

Comment:
Uncomment setup

Reply:
Good catch! Fixed. I think its best to leave the two references to enable_dropout hardcoded to false in decode - for decoding there is no reason to use dropout (it is only a useful featuring for training)